documentation-site/examples/spinner/determinate.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import * as React from 'react';
+import {Button} from 'baseui/button';
+import {SpinnerDeterminate} from 'baseui/spinner';
+import {useStyletron} from 'baseui';
+
+export default () => {
+  const [css] = useStyletron();
+  // Mimic some loading
+  const [x, xx] = React.useState(0);
```

Comment:
`x`, `xx` and `foo`, what am I seeing here haha?

Reply:
whoops - forgot to rename those 😅